### PR TITLE
Pull in apis and conformance tests from knative.dev/serving

### PR DESCRIPTION
**Pull request by tcnghia**
This PR only copies the whole history.  Modifications will follow.

### CODE REVIEW

File: pkg/apis/networking/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- networking-approvers
```
Comment (mattmoor) [inline pkg/apis/networking/OWNERS:4]: Do these existing in the repo root?

File: pkg/apis/networking/v1alpha1/serverlessservice_types.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/kmeta"
+	networking "knative.dev/serving/pkg/apis/networking"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_types.go:25]: I'd change the import paths before merging.

File: pkg/apis/networking/OWNERS (hunk 1 above)
Comment (tcnghia) [reply pkg/apis/networking/OWNERS]: Deleted the file to inherit from root.

File: pkg/apis/networking/v1alpha1/serverlessservice_types.go (hunk 2 above)
Comment (tcnghia) [reply pkg/apis/networking/v1alpha1/serverlessservice_types.go]: Done.

File: pkg/apis/networking/v1alpha1/serverlessservice_types.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_types.go:21]: Format Go code:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	networking "knative.dev/networking/pkg/apis/networking"
```

File: pkg/apis/networking/v1alpha1/serverlessservice_types.go [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/kmeta"
+	networking "knative.dev/networking/pkg/apis/networking"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_types.go:25]: Format Go code:
```suggestion

```

File: pkg/apis/networking/v1alpha1/serverlessservice_validation.go [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"knative.dev/pkg/apis"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation.go:23]: Format Go code:
```suggestion

```

File: pkg/apis/networking/v1alpha1/serverlessservice_validation.go [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+	"knative.dev/networking/pkg/apis/serving"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation.go:24]: Format Go code:
```suggestion
	"knative.dev/networking/pkg/apis/serving"
	"knative.dev/pkg/apis"
```

File: pkg/apis/networking/v1alpha1/serverlessservice_validation_test.go [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+package v1alpha1
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation_test.go:25]: Format Go code:
```suggestion

```

File: pkg/apis/networking/v1alpha1/serverlessservice_validation_test.go [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+	networking "knative.dev/networking/pkg/apis/networking"
```
Comment (mattmoor) [inline pkg/apis/networking/v1alpha1/serverlessservice_validation_test.go:26]: Format Go code:
```suggestion
	networking "knative.dev/networking/pkg/apis/networking"
	"knative.dev/pkg/apis"
```

File: test/conformance/certificate/utils.go [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/pkg/test/logging"
```
Comment (mattmoor) [inline test/conformance/certificate/utils.go:33]: Format Go code:
```suggestion

```

File: test/conformance/certificate/utils.go [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+	"knative.dev/networking/pkg/apis/networking"
+	"knative.dev/networking/pkg/apis/networking/v1alpha1"
```
Comment (mattmoor) [inline test/conformance/certificate/utils.go:35]: Format Go code:
```suggestion
	"knative.dev/networking/pkg/apis/networking/v1alpha1"
	"knative.dev/pkg/test/logging"
```

File: test/conformance/ingress/util.go [hunk 11]
```diff
@@ -0,0 +1,947 @@
...
+	"strings"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
```
Comment (mattmoor) [inline test/conformance/ingress/util.go:49]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/util/wait"
	"knative.dev/networking/pkg/apis/networking"
	"knative.dev/networking/pkg/apis/networking/v1alpha1"
```

File: test/conformance/ingress/util.go [hunk 12]
```diff
@@ -0,0 +1,947 @@
...
+	"knative.dev/pkg/network"
+	pkgTest "knative.dev/pkg/test"
+	"knative.dev/networking/pkg/apis/networking"
+	"knative.dev/networking/pkg/apis/networking/v1alpha1"
```
Comment (mattmoor) [inline test/conformance/ingress/util.go:53]: Format Go code:
```suggestion

```

**Comment by tcnghia** [comment]
Going to Rebase & Merge

**Pull request merged**